plugins/movetoarchive.koplugin/_meta.lua

```diff
@@ -1,6 +1,6 @@
 local _ = require("gettext")
 return {
-    name = "move_to_archive",
+    name = "movetoarchive",
```

Comment:
The before seems preferable to me? But yeah, I'd overlooked the move2archive one.

Reply:
We have no plugin with `_` in the directory name.
Could have been "move2archive", as we have already "send2ebook", but well, not minding.

Reply:
Oh, that was the problem? My bad, I interpreted it as just the mismatch between the metadata name and the widget name.